src/Modules/CalcPerform.lua

```diff
@@ -863,6 +863,7 @@ local function doActorMisc(env, actor)
 		end
 		if modDB:Flag(nil, "Condition:CanGainRage") or modDB:Sum("BASE", nil, "RageRegen") > 0 then
 			output.MaximumRage = modDB:Sum("BASE", skillCfg, "MaximumRage")
+			modDB.multipliers["MaxRageVortexSacrifice"] = output.MaximumRage / 4
```

Comment:
Poedb says that you can sacrifice 20% of rage, not 25%: https://poedb.tw/us/Rage_Vortex Also, shouldn't this just be a normal multiplier mod as well, or is this not possible due to the evaluation order?

Reply:
Update: This is support for helmet enchantment. Thanks, LI.